### DISCUSSION TITLE
Remove friends and maybe a way to get online ID using account ID

### DISCUSSION THREAD

**Discussion by Kenshin9977** [isFakeAccount/psnawp#8] [Ideas]
What I want is a way to delete multiple friends automatically. I tried using Selenium but to no avail. This API seems to be the way to proceed.
What would be awesome is the possibility to delete a friend through the API as it would allow me to do what I want. Through Selenium I'm limited to 4 deletion before the website ignores my actions. Is it doable ?
With this API I can also get my friends list but only their account ID. To sort them I'd need a way to get their online ID directly or using their account ID. At worst I can manage some other way to retrieve the friends list with their online ID through Selenium.

**Comment by isFakeAccount** [comment]
I can look into the deletion endpoint if it is available. I think there is but I can't guarantee if it will work.

**Comment by isFakeAccount** [comment]
@Kenshin9977 v2.1.0 now implements this endpoint.
